ui/src/app/App.js

```diff
@@ -123,6 +123,7 @@ export const App = () => {
         )}
         location={location}
         logout={() => {
+          dispatch(statusActions.websocketDisconnect());
```

Comment:
This is a drive-by to disconnect from the websocket when logging out. This ws connection should not be left open if you're not  authenticated.